docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+
+Now, we can use GeometryOperations and CoordinateTransformations to shift `polygon1`
+up, to avoid plotting over our earlier results.  This is done through the [GeometryOps.transform](@ref) function.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+Polygons can contain "holes". The first `LinearRing` in a polygon is the exterior, and all 
+subsequent `LinearRing`s are treated as holes in the leading `LinearRing`.
+
+`GeoInterface` offers the `GI.getexterior(poly)` and `GI.gethole(poly)` methods to get the 
+exterior ring and an iterable of holes, respectively.
+
+!!! note
+    Some packages always consider the secondary `LinearRings` holes, others look at the winding
+    order, where the polygons are filled inward if they have a [clockwise](@ref GeometryOps.isclockwise) winding order and 
+    outward if they have a counterclockwise winding order.
+    
+    Hopefully, these are details that you'll never have to deal with.  But it is good to know.
+
+````@example creating_geometry
+hole = GI.LinearRing(GI.getpoint(multipoint))
+polygon1 = GI.Polygon([ring1, hole]; crs)
+````
+
+Shift `polygon1` to the right, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 50.
+yoffset = 0.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+`Polygon`s can also be grouped together as a `MultiPolygon`.
+
+````@example creating_geometry
+r = 5
+x = cos.(reverse(ϴ)) .* r .+ xoffset
+y = sin.(reverse(ϴ)) .* r .+ yoffset
+ring2 =  GI.LinearRing(GI.Point.(zip(x,y)))
+polygon2 = GI.Polygon([ring2])
+multipolygon = GI.MultiPolygon([polygon1, polygon2]; crs)
+````
+
+Shift `multipolygon` up, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+multipolygon = GO.transform(f, multipolygon)
+plot!(multipolygon)
+fig
+````
+
+Great, now we can make `Points`, `MultiPoints`, `Lines`, `LineStrings`, `Polygons` (with holes), and `MultiPolygons` and modify them using [`CoordinateTransformations`] and [`GeometryOps`].
+
+But where does the `crs` information come in? To show this, we need to use `GeoMakie` that can interpret the `crs` information that we've included with our geometries.
+
+Now specify the source and destination projections for our map. Remember that the very first thing we did was select our source coordinate system.
+
+````@example creating_geometry
+source = crs;
+dest = "+proj=natearth2" #see [https://proj.org/en/9.4/operations/projections/natearth2.html]
+````
+
+Open the Natural Earth continental outlines that are bundled with GeoMakie, and also available from https://www.naturalearthdata.com/.
+
+````@example creating_geometry
+land_path = GeoMakie.assetpath("ne_110m_land.geojson")
```

Comment:
What is this particular land path? I am left wondering what `ne_110m_land` means.

Reply:
This is the natural earth 110m land polygons.  We could download this straight from NaturalEarth and add a comment about GeoMakie also shipping it.  

An explanation might look like:
```md
Download and open the [Natural Earth](https://www.naturalearthdata.com) global land polygons at 110m resolution.  This will serve as our background image to add context to the map.

```